src/factories/store/factoriesSelectors.ts

```diff
@@ -35,6 +35,24 @@ export const useFactorySimpleAttributes = (id: string | null | undefined) => {
       id: factory?.id,
       name: factory?.name,
       description: factory?.description,
+      progress: factory?.progress,
+      boardIndex: factory?.boardIndex,
+    };
+  });
+};
+
+export const useSafeFactorySimpleAttributes = (id?: string) => {
```

Comment:
Not sure about the naming but I don't think this should be an issue

Reply:
What I didn't want was a factory being not found then leading to undefined attributes. And then not being able to know whether the factory existed but without said attributes or the factory doesn't exist. I ended up not using it so I can remove it for now.